docs/about/products.md

```diff
@@ -0,0 +1,72 @@
+
+Major commercial and open source .NET products are listed below, in order of introduction.
+
+.NET Framework
+--------------
+
+The .NET Framework is Microsoft's primary develop platform for Windows server and client developers. It is a very powerful and mature framework, with a large class library that supports a wide variety of applications and solutions on Windows.
+
+There are multiple application models built on top of the .NET Framework that allow developers to build applications ranging from console applications, across rich client (WPF) applications to scalable web applications.
+
+[Windows Forms](https://msdn.microsoft.com/library/dd30h2yb.aspx) and [Windows Presentation Foundation (WPF)](https://msdn.microsoft.com/library/ms754130.aspx) are User Interface (UI) stacks that enable you to build desktop applications for Windows. The strength of Windows Forms lies in its rich support for common databinding scenarios as well as access to Windows’ native user interface controls. WPF, on the other hand, allows you to exercise much more control over the look and feel of your application. 
+
+[Windows Communication Foundation (WCF)](https://msdn.microsoft.com/library/ms731082.aspx) is a set of libraries for SOAP Web Services. It allows you to create services that can communicate through various supported protocols using various data formats, and that can be hosted in any process you choose. This leads to one of the major features of WCF: your services are not tied to any particular hosting strategy or approach.
+
+[ASP.NET](http://www.asp.net/) is a web framework. Being a very rich framework, it has several distinct pieces which are used to produce modern and high-performance web applications. [ASP.NET Web Forms](http://www.asp.net/web-forms) is a set of tools geared primarily towards developer productivity, allowing quick turnaround on web applications with a drag-and-drop surface reusing web controls for everything from logging to data binding. [ASP.NET MVC](http://www.asp.net/mvc) gives you greater control over the entire web pipeline, from the HTTP layer to the user interface. [ASP.NET WebAPI](http://www.asp.net/web-api) is a convention-based framework for creating REST services. Finally, [SignalR](http://www.asp.net/signalr) allows you to provide push-based communication to your web applications using the [WebSocket](https://en.wikipedia.org/wiki/WebSocket) protocol.
+
+Mono
+----
+
+[Mono](http://www.mono-project.com/docs/about-mono/) is the original open source and cross-platform implementation of .NET, from the community [Mono Project](http://mono-project.com). It is now sponsored by Microsoft. It can be thought of as an open and cross-platform version of the .NET Framework. It's APIs follow the progress of the .NET Framework, not .NET Core.
```

Comment:
"follow ... .NET FX, not .NET Core" -- the text hasn't mentioned .NET Core yet.  Maybe a line or two at the top would make sense to provide some context: "The major  .NET products are:  .... They are listed below, in order of introduction."